python/doc/theory/data_analysis/quantile_asymptotic_confidence_estimation.rst

```diff
@@ -0,0 +1,100 @@
+.. _quantile_asymptotic_confidence_estimation:
+
+Asymptotic quantile confidence interval based on order statistics
+-----------------------------------------------------------------
+
+We consider a random variable :math:`X` of dimension 1 and its quantile :math:`x_{\alpha}`
+of level :math:`\alpha` (:math:`\alpha \in [0, 1]`).
+We want to determine an asymptotic confidence interval of :math:`x_{\alpha}` with a confidence greater or equal to
+:math:`\beta`, using order statistics.
+
+Let :math:`(X_1, \dots, X_\sampleSize)` be some independent copies of :math:`X`.
+
+Let :math:`(X_1, \dots, X_\sampleSize)` be some independent copies of :math:`X`.
+Let :math:`X_{(k)}` be the :math:`k` -th order statistics of :math:`(X_1, \dots, X_\sampleSize)` which means that
+:math:`X_{(k)}` is the :math:`k` -th minimum of :math:`(X_1, \dots, X_\sampleSize)` for :math:`1 \leq k \leq \sampleSize`. For
+example, :math:`X_{(1)} = \min (X_1, \dots, X_\sampleSize)` is the minimum
+and :math:`X_{(\sampleSize)} = \max (X_1, \dots, X_\sampleSize)` is the maximum.
+
+Empirical quantile estimator
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+We first introduce the empirical estimator of the quantile :math:`x_{\alpha}`.
+We denote by :math:`\hat{F}` the empirical cumulative distribution function defined by:
+
+ .. math::
+
+        \hat{F}(x) = \dfrac{1}{\sampleSize} \sum_{i=1}^\sampleSize 1_{X_i \leq x}
```

Comment:
<img width="1272" height="251" alt="image" src="https://github.com/user-attachments/assets/063a361a-c8f1-48fc-91cb-8612ee70abee" />
This should work as expected, using a correct typographic font.

```suggestion
        \hat{F}(x) = \dfrac{1}{\sampleSize} \sum_{i=1}^\sampleSize \mathbb{1}_{X_i \leq x}
```